podman/api_connection.py

```diff
@@ -87,22 +94,16 @@ def post(self, path, params=None, headers=None):
                             body=data,
                             headers=headers)
 
-    def request(self,
-                method,
-                url,
-                body=None,
-                headers=None,
-                *,
-                encode_chunked=False):
+    def request(
+        self, method, url, body=None, headers=None, *, encode_chunked=False
+    ):
         """Make request to Podman service."""
         if headers is None:
             headers = {}
 
-        super().request(method,
-                        url,
-                        body,
-                        headers,
-                        encode_chunked=encode_chunked)
+        super().request(
+            method, url, body, headers, encode_chunked=encode_chunked
```

Comment:
I think we should keep it aligned with the opening delimiter or if you are going to do this, I think the ) shouldn't be separated on a new line. Seems more in line with https://www.python.org/dev/peps/pep-0008/#indentation